src/app/pages/applications/catalog/catalog.component.ts

```diff
@@ -311,7 +311,7 @@ export class CatalogComponent implements OnInit {
           '500px', 'info', true);
       });
     });
-    dialogRef.componentInstance.failure.pipe(untilDestroyed(this)).subscribe((err: string) => {
+    dialogRef.componentInstance.failure.pipe(untilDestroyed(this)).subscribe((err) => {
```

Comment:
Shouldn't err also be typed?

Reply:
It's typed. It just gets its type from:
```
src/app/pages/common/entity/entity-job/entity-job.component.ts:44
```
Our ts settings don't allow for not having typing. It won't compile and require at least 'any' put it if it cannot infer the type. In this case it can, because `failure` is typed now.